examples/gc.ml

```diff
@@ -99,11 +99,19 @@ let rec megabytes_of_path path =
 let gc_all_but_head repo branch =
   let* head = Store.Head.get branch in
   let head_key = Store.Commit.key head in
+  let module S = Irmin_pack_unix.Stats.Latest_gc in
   let finished = function
-    | Ok (r : Store.Gc.stats) ->
+    | Ok (r : S.stats) ->
+        let steps = r.steps |> List.map (fun (k, v) -> (k, v.S.wall)) in
+        let duration = steps |> List.map snd |> List.fold_left Float.add 0. in
+        let finalisation_duration =
+          duration
+          -. List.assoc "worker startup" steps
+          -. List.assoc "before finalise" steps
+        in
```

Comment:
I want to strongly advocate for keeping the simplified stats in the gc finished callback (and probably the entire stack of finalised stats passing). This level of complexity (and internal details knowledge needed) shouldn't be in the public api.

Reply:
I can tackle this by adding getters in `Stats.Latest_gc` that decipher the stats

Reply:
I think that would work.

Reply:
Done. See latest commit